### PR TITLE
Homescreen – Extend UI of "Risiko-Ermittlung aktiv"

### PR DESCRIPTION
**Pull request by IonutMacovei**
## Checklist

* [x] This PR does not change text that hasn't been signed off with the RKI. Have a look at the pinned issue [440](https://github.com/corona-warn-app/cwa-app-ios/issues)
* [ ] Test your changes as thoroughly as possible before you commit them. Preferably, automate your test by unit/integration tests.
* [ ] If this PR comes from a fork, please [Allow edits from maintainers](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)
* [ ] Set a speaking title. Format: {task_name} (closes #{issue_number}). For example: Use logger (closes # 41)
* [x] [Link your Pull Request to an issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) - Pull Requests that are not linked to an issue with you as an assignee will be closed.
* [ ] Create Work In Progress [WIP] pull requests only if you need clarification or an explicit review before you can continue your work item.
* [x] Make sure that your PR is not introducing _unnecessary_ reformatting (e.g., introduced by on-save hooks in your IDE)

## Description

This PR is related to https://jira.itc.sap.com/browse/EXPOSUREAPP-2486

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Views/Home Screen/Cells/ActivateCollectionViewCell.swift [hunk 1]
```diff
@@ -29,8 +31,14 @@ class ActivateCollectionViewCell: HomeCardCollectionViewCell {
 		accessibilityTraits = .button
 	}
 
-	func configure(title: String, icon: UIImage?, animationImages: [UIImage]? = nil, accessibilityIdentifier: String) {
+	func configure(title: String, subtitle: String? = nil, icon: UIImage?, animationImages: [UIImage]? = nil, accessibilityIdentifier: String) {
 		self.titleLabel.text = title
+		self.subtitleLabel.text = subtitle
+		if self.subtitleLabel.isTruncated {
```
Comment (melloskitten) [inline src/xcode/ENA/ENA/Source/Views/Home Screen/Cells/ActivateCollectionViewCell.swift:37]: can be shortened to

```swift
subTitleLabel.text = subtitleLabel.isTruncated ? AppStrings.ExposureNotificationSetting.euWideSubtitle : subtitle
```

File: src/xcode/ENA/ENA/Source/Views/Home Screen/Cells/ActivateCollectionViewCell.swift [hunk 2]
```diff
@@ -29,8 +31,14 @@ class ActivateCollectionViewCell: HomeCardCollectionViewCell {
...
+			self.subtitleLabel.text = AppStrings.ExposureNotificationSetting.euWideSubtitle
```
Comment (melloskitten) [inline src/xcode/ENA/ENA/Source/Views/Home Screen/Cells/ActivateCollectionViewCell.swift:38]: remove all unneeded `self` from this method

**Review by melloskitten — CHANGES_REQUESTED** [review]
Hi there, thank you very much for your contribution! I have added some comments to the PR 👍

File: src/xcode/ENA/ENA/Source/Views/Home Screen/Cells/ActivateCollectionViewCell.swift (hunk 2 above)
Comment (IonutMacovei) [reply src/xcode/ENA/ENA/Source/Views/Home Screen/Cells/ActivateCollectionViewCell.swift]: Done

File: src/xcode/ENA/ENA/Source/Views/Home Screen/Cells/ActivateCollectionViewCell.swift (hunk 1 above)
Comment (IonutMacovei) [reply src/xcode/ENA/ENA/Source/Views/Home Screen/Cells/ActivateCollectionViewCell.swift]: Done

**Pull request closed without merging**